### PR TITLE
Release 4.11.1

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-release-4.11.1](https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-release-4.11.1)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 4.11.1 (2021-05-20)
+
+- Fixed the date range picker to disable the inner form controls when consumers use a reactive form control during initialization or the `disabled` input. [#261](https://github.com/blackbaud/skyux-datetime/pull/261)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md:3]:
```suggestion
- Fixed the date range picker to disable the inner form controls when consumers use reactive form controls during initialization and when they use the `disabled` input. [#261](https://github.com/blackbaud/skyux-datetime/pull/261)
```

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-release-4.11.1](https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-release-4.11.1)

**Review by blackbaud-johnly — APPROVED** [review]

**Review by blackbaud-johnly — APPROVED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-release-4.11.1](https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-release-4.11.1)

**Pull request merged**